api_sentry.py

```diff
@@ -0,0 +1,201 @@
+import os
+import sys
+
+import pandas as pd
+
+from lib.sentry_conn import APIClient
+
+from database import (
+    Database,
+    ReportSentryIssues
+)
+
+# The 2 major versions are beta and release.
+NUM_MAJOR_VERSIONS = 2
```

Comment:
When we talk to Nish, let's make sure this would be enough or if they would like to monitor more versions